.phpcs.xml.dist

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <ruleset name="Jetpack">
-	<config name="minimum_supported_wp_version" value="4.8" />
+	<config name="minimum_supported_wp_version" value="4.9" />
```

Comment:
Good call. Let's do the same in the readme, we had not done that yet.
ac37c149dbae306b4fe82d629304604ccf03cac5